src/System.Net.Sockets/System.Net.Sockets.sln

```diff
@@ -57,10 +57,10 @@ Global
 		{9E212427-18B3-4EF4-966C-ED18AAC08422}.Windows_Release|Any CPU.Build.0 = Debug|Any CPU
 		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.Debug|Any CPU.ActiveCfg = Windows_Debug|Any CPU
 		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.Debug|Any CPU.Build.0 = Windows_Debug|Any CPU
-		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Debug|Any CPU.ActiveCfg = FreeBSD_Debug|Any CPU
-		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Debug|Any CPU.Build.0 = FreeBSD_Debug|Any CPU
-		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Release|Any CPU.ActiveCfg = FreeBSD_Release|Any CPU
-		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Release|Any CPU.Build.0 = FreeBSD_Release|Any CPU
+		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Debug|Any CPU.ActiveCfg = Windows_Release|Any CPU
+		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Debug|Any CPU.Build.0 = Windows_Release|Any CPU
+		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Release|Any CPU.ActiveCfg = Windows_Release|Any CPU
+		{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}.FreeBSD_Release|Any CPU.Build.0 = Windows_Release|Any CPU
```

Comment:
I see below that I'm actually wrong: the csproj files still have FreeBSD.  @mellinoe Maybe we should try to remove it from this CSProj too to avoid this issue?